topi/tests/python_cpp/test_topi_tensor.py

```diff
@@ -0,0 +1,127 @@
+"""Test code for tensor operator"""
+import numpy as np
+import tvm
+import topi
+
+def verify_elemwise_sum(num_args, dtype):
```

Comment:
Remove num_args here as well.

Reply:
I use num_args here to generate test data for doing various element_wise tests.